backend/order/views.py

```diff
@@ -1,3 +1,13 @@
-# from django.shortcuts import render
+from order.models import Order
+from rest_framework import generics
+from .serializers import OrderSerializer
 
-# Create your views here.
+
+class ListAllRequisitions(generics.ListAPIView):
+    serializer_class = OrderSerializer
+    queryset = Order.objects.all()
```

Comment:
El nombre y lo que hace la funcion es diferente. Si lo que querias lograr es enlistar las ordenes entonces seria algo asi.

```suggestion
class ListOrders(generics.ListAPIView):
    serializer_class = OrderSerializer
    queryset = Order.objects.all()
```